### DISCUSSION TITLE
About DIY marker Icon

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#55] [Ideas]
Can I use my own PNG image format marker icon?

**Comment by valentine195** [comment]
Not yet, but I think I could add this feature. I will add an issue to track it. No timeline on it, sorry.

**Reply by zhubib1234** [reply]
Next，I'm looking forward to this function.

**Reply by valentine195** [reply]
Added in 3.12.0

Marker icons can be set to images in settings.

Please note that images will be resized to 24px wide (same width as current marker icons).

**Reply by zhubib1234** [reply]
Great. Love you.

**Comment by zhubib1234** [comment]
The icon background is black, opaque, GIF, PNG format are opaque. How to set it? thank you!

**Comment by zhubib1234** [comment]
![ca](https://user-images.githubusercontent.com/84388916/119573733-40f7aa80-bde7-11eb-933c-8d97f6632caf.png)
![ca](https://user-images.githubusercontent.com/84388916/119573758-48b74f00-bde7-11eb-9eca-1eec4b6e8b0f.gif)
![008](https://user-images.githubusercontent.com/84388916/119573773-4f45c680-bde7-11eb-8d47-c9a99ed5a691.JPG)
![009](https://user-images.githubusercontent.com/84388916/119573802-540a7a80-bde7-11eb-8404-f1e08457c77c.JPG)

**Reply by valentine195** [reply]
Okay, I'll take a look - I wasn't seeing this behavior when testing.

**Comment by valentine195** [comment]
Please retry in 3.13.0.

You will have to re-upload the image.

**Reply by zhubib1234** [reply]
thanks！

**Comment by zhubib1234** [comment]
![KCMEZ61F6~CKEN(3C3Q3_}H](https://user-images.githubusercontent.com/84388916/119737141-62bd6400-beb1-11eb-9be8-803064ba530c.png)

**Reply by zhubib1234** [reply]
perfect
